src/components/Slider/README-ru.md

```diff
@@ -0,0 +1,285 @@
+| [min](#min-and-max-value)            | Минимальное значение компонента.                                                                                                                                       |                       `number`                       |          `0`          |
+| onBlur                               | Срабатывает, когда контрол теряет фокус. Передает событие фокуса в качестве аргумента обратного вызова.                                                                | `((e: FocusEvent<HTMLDivElement, Element>) => void)` |                       |
+| onUpdate                             | Срабатывает, когда пользователь изменяет значение слайдера. Передает событие изменения в качестве аргумента обратного вызова.                                          |   `((value: number \| [number, number]) => void)`    |                       |
+| onUpdateComplete                     | Активируется при срабатывании `ontouchend` или `onmouseup`. Передает событие изменения в качестве аргумента обратного вызова.                                          |   `((value: number \| [number, number]) => void)`    |                       |
+| onFocus                              | Срабатывает, когда контрол получает фокус. Передает событие фокуса в качестве аргумента обратного вызова.                                                              | `((e: FocusEvent<HTMLDivElement, Element>) => void)` |                       |
+| [size](#size)                        | Размер контрола.                                                                                                                                                       |               `"s"` `"m"` `"l"` `"xl"`               |         `"m"`         |
+| [step](#step)                        | Шаги приращения и уменьшения при перемещении слайдера. Свойство игнорируется, если задано `availablevalues`.                                                           |                       `number`                       |          `1`          |
+| tabIndex                             | Атрибут `tabIndex` для контрола.                                                                                                                                       |             `number` `[number, number]`              |                       |
+| [validationState](#error)            | Состояние валидации.                                                                                                                                                   |                     `"invalid"`                      |                       |
+| [value](#slider-variants)            | Значение контрола.                                                                                                                                                     |             `number` `[number, number]`              |                       |
```

Comment:
The same comment as for en version. We have newer version of readme(with new API) than the translated.

Reply:
Hi, fixed, please recheck!